MapboxCoreNavigation/OfflineDirections.swift

```diff
@@ -77,6 +82,39 @@ public class NavigationDirections: Directions, OfflineRoutingProtocol {
         }
     }
     
+    public class func unpackTilePack(at filePath: URL, outputDirectory: URL, progressHandler: UnpackProgressHandler?, completionHandler: UnpackCompletionHandler?) {
+        
+        OfflineDirectionsConstants.offlineSerialQueue.sync {
+            
+            let totalPackedBytes = filePath.fileSize()!
+            
+            // Report 0% progress
+            progressHandler?(totalPackedBytes, totalPackedBytes)
+
+            var timer: CountdownTimer? = CountdownTimer(countdown: .seconds(500), accuracy: .seconds(500), executingOn: OfflineDirectionsConstants.unpackSerialQueue, payload: {
```

Comment:
Also, why is the timer an optional here? AFAIK `CountdownTimer` doesn't have a bailable initializer.

Reply:
b/c I wanted to disarm it before nilling it out at the end. Not even keeping a reference could be an option? 🤔